docs/getting-started.rst

```diff
@@ -288,16 +288,16 @@ Let's start by bringing up a Prometheus instance ourselves, to scrape our applic
     # prometheus.yml
     scrape_configs:
     - job_name: 'my-app'
-        scrape_interval: 5s
-        static_configs:
-        - targets: ['localhost:8000']
+      scrape_interval: 5s
+      static_configs:
+      - targets: ['localhost:8000']
 
 And start a docker container for it:
 
 .. code-block:: sh
 
     # --net=host will not work properly outside of Linux.
-    docker run --net=host -v ./prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus\
+    docker run --net=host -v `pwd`/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus \
```

Comment:
thoughts on maybe just calling out /tmp/ as a directory here? I've found that little tricks like backticks and the pwd command get lost on people newer to linux or programming in general.

Reply:
You're right, another issue is that pwd is unix only. I updated it to `/tmp/`.